virttest/utils_test/__init__.py

```diff
@@ -989,6 +992,7 @@ def env_check(self):
                 logging.error("Failed to restart libvirtd inside guest")
                 return False
             pip_pack = ['setuptools', 'netifaces', 'aexpect', 'netaddr']
+            pip_pack.insert(0, self.pip_bin)
```

Comment:
This should simply be:

```python
pip_pack = [self.pip_bin, 'setuptools', 'netifaces', 'aexpect', 'netaddr']
```

Reply:
Actually why are you adding the `pip_bin` to `pip_pack`? `pip_pack` is a list of python packages to install/upgrade and the `pip_bin` is added on line 994/998.

Reply:
If you meant to update pip than you should simply hardcode `pip` and not `self.pip_bin` because the pip package is called `pip` for both, python2 and python3.

Reply:
@clebergnu @ldoktor 
Yes, actually I want to upgrade the 'pip', because we can have installed the old version of pip and exit code will be '0'
And Lukas apart of that I have some question to 'pip_pack'
Can we add to a list a 'configparser' package? Or we can add it to RHEL-7-devel.ks file?
In the RHEL-7 I got the next error:
"Unable to import Avocado libraries, please verify your installation, and if necessary reinstall it."

Reply:
@clebergnu @ldoktor 
Thanks a lot for your review! And sorry for my gaps =)